docs/sftt/sftt-api-spec.md

```diff
@@ -333,6 +357,8 @@ If the calling user is not team leader.
 
 Invite someone to join a team. Will send an email to all specified people that includes a link. Link will direct them to the team page where they can join once they are authenticated. If one of the email addresses is invalid or the user is already on the team that invite will not be send out, the other ones will be and a `200 OK` response is returned.
 
+!!! Still in progress, emails are currently not sent.
```

Comment:
❓ Is this an issue with the emailer or just not implemented?

Reply:
Just not implemented, as far as I know the email server is still working for the forgot password requests.